modules/ROOT/pages/dataweave-cookbook-decode-encode.adoc

```diff
@@ -0,0 +1,158 @@
+= Decode and Encode Base64
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, xml, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
+
+The DataWeave examples show how to convert a file stream into Base64 and to convert a Base64 string into a file stream. They use a PDF file as the input and output.
+//LINK TO DW 1.0 EXAMPLES:
+include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
+
+[[xml_config]]
+== XML Configuration Example
```

Comment:
Reading this first made me confused about whether the example was done and what the other sections were about. I'd move this to the bottom as a full Mule app example featuring the functionality in the previous examples that you can try in Studio.

Reply:
moved to end